README.md

```diff
@@ -204,6 +204,11 @@ The low-level API is as follows:
 
 For examples how to use them, please refer to the README of the above mentioned solc-js releases.
 
+**Note**: These low-level functions remain available for compatibility reasons.
+However, they were superseded by the `compile()` function and are no longer required.
+Starting in version `0.5.0+commit.1d4f565a`, the functions `compileSingle`, `compileMulti`, and `compileCallback` are always `null` when using newer binary versions.
+It is recommended to use the latest release of solc-js, but it should also handle all the older binaries down to `0.1.x`.
```

Comment:
```suggestion
It is recommended to use the latest release of solc-js, but it should also handle all the older solc binaries down to `0.1.x`.
```
This feels confusing to me too, who should handle older binaries? `solc` or `solc-js` binaries?

Reply:
The `solc-js` should handle old `solc` binaries.